api/app/signals/apps/signals/models/source.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.gis.db import models
+
+
+class Source(models.Model):
+    name = models.CharField(max_length=128)
```

Comment:
Maybe add a description, so that a bit more information can be provided about the source.